app/controllers/grade_entry_forms_controller.rb

```diff
@@ -414,8 +425,36 @@ def csv_upload
           end
         end
       end
+      # Display a warning message if the newly uploaded file
+      #      overrides existing collumn names and/or grade totals
+      matching_names = true
+      begin
+        if !abort_upload
+          grade_entry_items =
+            GradeEntryItem.where(grade_entry_form_id: @grade_entry_form.id)
+          lines = File.foreach(params[:upload][:grades_file].path.to_s).first(2)
+          uploaded_column_names =  lines[0].split(',').map(&:strip)
+          uploaded_grade_totals = lines[1].split(',').map(&:strip)
+          # Starting with index 1 as we wish to skip the first empty csv cell
+          count = 1
+          grade_entry_items.each do |original|
+            unless uploaded_column_names[count] == original.name &&
+              uploaded_grade_totals[count] == original.out_of.to_i.to_s
```

Comment:
Align the operands of a condition in an `unless` statement spanning multiple lines.